### PR TITLE
Devops and security tools

### PR DESCRIPTION
**Pull request by gngster**
just added devops and sec tools that i researched on and think that we should use

### CODE REVIEW

File: tool_assessment.md [hunk 1]
```diff
@@ -52,4 +52,16 @@ AES [example](https://aesencryption.net/).
 
 ## Source Code Control & Issue Management
 
-Our project uses [GitHub](https://github.com/IFS4205-2018-Sem1-Team1) for source code control and issue management. Reports will be written in .md (instead of .doc) to allow for version control of reports as well.
\ No newline at end of file
+Our project uses [GitHub](https://github.com/IFS4205-2018-Sem1-Team1) for source code control and issue management. Reports will be written in .md (instead of .doc) to allow for version control of reports as well.
+
+## DevOps Tools
+
+Our project will use [Ansible](https://www.ansible.com/) to ensure that our remote servers are all set up with the same configuration.
```
Comment (chilipadiboy) [inline tool_assessment.md]: wew ansible. From what I read it is just a deployment tool. Not sure how to use it :(

File: tool_assessment.md [hunk 2]
```diff
@@ -52,4 +52,16 @@ AES [example](https://aesencryption.net/).
...
\ No newline at end of file
+Our project uses [GitHub](https://github.com/IFS4205-2018-Sem1-Team1) for source code control and issue management. Reports will be written in .md (instead of .doc) to allow for version control of reports as well.
+
+## DevOps Tools
+
+Our project will use [Ansible](https://www.ansible.com/) to ensure that our remote servers are all set up with the same configuration. 
+
+## Security Tools
+
+Our project will use [Metaspolit](https://www.metasploit.com/) and [Arachni](http://www.arachni-scanner.com/) to scan our web application, as well as [phpcs-security-audit](https://github.com/FloeDesignTechnologies/phpcs-security-audit) to scan our PHP code for vulnerabilities. We would also use [Synk](https://github.com/Snyk/) to integrate easily with our Github repo find & fix known vulnerabilities in open-source dependencies.  We can also consider using [BDD-Security](https://www.continuumsecurity.net/bdd-security/) to launch automated scans with specific scenarios/ claims we want to keep.
```
Comment (chilipadiboy) [inline tool_assessment.md]: can add CIS-CAT scanner for server configuration? https://learn.cisecurity.org/cis-cat-landing-page

I will get this soon.

**Comment by Zhiyuan-Amos** [comment]
Happened to see a bit more tools on past week's presenter's slides:
1. [Sentry](https://sentry.io/welcome/)
1. [Checkmarx](https://www.checkmarx.com/)

Are these relevant?

File: tool_assessment.md (hunk 2 above)
Comment (Zhiyuan-Amos) [reply tool_assessment.md]: @gngster will you be addressing this?

**Pull request merged**